sktime/utils/validation/panel.py

```diff
@@ -70,9 +72,11 @@ def check_X(
     # check first if we have the right number of dimensions, otherwise we
     # may not be able to get the shape of the second dimension below
     if isinstance(X, np.ndarray):
-        if not X.ndim == 3:
+        if X.ndim == 2:
```

Comment:
If we're reshaping 2d arrays now doesn't it just make sense to reshape 1d arrays as well?

i.e.
if X.ndim == 1:
    X = X.reshape(X.shape[0], 1, 1)

Reply:
hmmm, maybe, that would make sense for predict and predict_proba, but not for fit, doesnt make sense to build a classifier on one case. But lets stick to this for now :)